### PR TITLE
add a panacea(MED) chain

### PR DESCRIPTION
**Pull request by gyuguen**

### CODE REVIEW

File: panacea/chain.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    "chain_name": "panacea",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Panacea",
+    "chain_id": "panacea-3",
+    "bech32_prefix": "panacea",
+    "daemon_name": "panacead",
+    "node_home": "$HOME/.panacead",
+    "genesis": {
+        "genesis_url": "https://github.com/medibloc/panacea-mainnet/raw/master/panacea-3/genesis.json.gz"
```
Comment (sunnya97) [inline panacea/chain.json:12]: Can you swap this for an un-encrypted json?  I suggest using git lfs to host the unencrypted genesis file on github.

Comment (gyuguen) [reply panacea/chain.json:12]: Okay. I'll modify it when this [pr](https://github.com/medibloc/panacea-mainnet/pull/24) is merge.

Comment (gyuguen) [reply panacea/chain.json]: I modified the path of genesis.json.
https://github.com/cosmos/chain-registry/pull/49/commits/a354081a941e3db5574461889bc69f070e21d67d

**Comment by sunnya97** [comment]
Thank you!

**Pull request merged**